next.config.js

```diff
@@ -25,3106 +25,8 @@ const nextConfig = {
   },
 
   redirects() {
```

Comment:
I think we can remove the redirects function completely

Reply:
Yes, we can.

I kept it there to signal the intention of having no redirects in the config

Reply:
I'd rather remove it. Keeping it around seems unnecessary. People are still allowed to define static redirects.